packages/teleport/src/Console/DocumentSsh/DocumentSsh.tsx

```diff
@@ -15,16 +15,21 @@ limitations under the License.
 */
 
 import React, { useRef, useEffect } from 'react';
-import cfg from 'teleport/config';
+
 import * as Icons from 'design/Icon';
 import { Indicator, Text, Box, ButtonPrimary } from 'design';
 import * as Alerts from 'design/Alert';
+
+import cfg from 'teleport/config';
 import * as stores from 'teleport/Console/stores';
-import FileTransfer, { useFileTransferDialogs } from './../FileTransfer';
-import Terminal from './Terminal';
-import Document from '../Document';
+
 import AuthnDialog from 'teleport/components/AuthnDialog';
 import useWebAuthn from 'teleport/lib/useWebAuthn';
+
+import Document from '../Document';
+
+import Terminal from './Terminal';
+import FileTransfer, { useFileTransferDialogs } from './../FileTransfer';
```

Comment:
goupring error

Reply:
This is correct grouping as per the rules, but it's confusing the linter because of the `./`. So it's grouping it with `./Terminal` instead of `../Document`.